tests/subworkflows/nf-core/bam_stats_samtools/test.yml

```
@@ -1,13 +1,14 @@
 - name: bam stats samtools single-end
-  command: nextflow run ./tests/subworkflows/nf-core/bam_stats_samtools -entry test_bam_stats_samtools_single_end -c tests/config/nextflow.config
+  command: nextflow run ./tests/subworkflows/nf-core/bam_stats_samtools -entry test_bam_stats_samtools_single_end -c ./tests/config/nextflow.config -c ./tests/subworkflows/nf-core/bam_stats_samtools/nextflow.config
   tags:
```

```suggestion
  tags: &tags
```

We could use an anchor here to avoid copying and pasting the tags over and over, but that's probably going to make linting harder.